server/liveblog/posts/posts.py

```diff
@@ -26,7 +26,8 @@ def private_draft_filter():
     """
     private_filter = {'should': [{'term': {'post_status': 'open'}},
                                  {'term': {'post_status': 'submitted'}},
-                                 {'term': {'post_status': 'comment'}}]}
+                                 {'term': {'post_status': 'comment'}},
```

Comment:
Very strange to have 'comment' as a possible status for a post, no? Is 'comment' a status like 'open' or 'submitted'? Seems more like a type.